src/View/Helper/PaginatorHelper.php

```diff
@@ -1169,4 +1170,38 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param int $default Default limit for option selecting. Default is Paginator limit.
+     * @param array $options Options for Select tag attributes like class, id or event
+     *
+     * @return string html output.
+     */
+    public function limitSelect($limits = [], $default = null, $options = [])
+    {
+        $out = $this->Form->create(null, ['type' => 'get']);
+
+        if (empty($limits)) {
+            $limits = [10 => 10, 25 => 25, 50 => 50, 100 => 100];
+        }
+
+        if (empty($default) || !is_integer($default)) {
+            $default = $this->param('perPage');
+        }
+
+        if (!in_array($default, $limits)) {
+            $limits += [$default => $default];
+        }
+
+        ksort($limits);
+
+        $out .= $this->Form->input('limit', ($options + ['type' => 'select', 'label' => __("View") . '&nbsp;', 'escape' => false, 'value' => $default, 'default' => $default, 'options' => $limits, 'onChange' => 'this.form.submit()']));
```

Comment:
I don't think `'escape' => false` is necessary, also `'label' => __('View')` would probably be a better.